libs/mini-rx-store/src/lib/component-store.ts

```diff
@@ -53,6 +52,7 @@ export class ComponentStore<StateType extends object>
 
         const metaReducers: MetaReducer<StateType>[] = [];
 
+        // TODO refactor extensions setup: use `calculateExtensions`
```

Comment:
@carlos-ds this can be done as preparation of the inheritance refactor.

Then the component store will be a bit thinner already.